client/src/containers/App/App.js

```diff
@@ -40,6 +45,7 @@ const App = () => (
       <Route path="/shelters" component={Shelters} />
       <Route path="/referrals" component={Referrals} />
       <Route path="/users" component={Users} />
+      <Route path="/VolunteersForm" component={VolunteersForm} />
```

Comment:
..if you change the above, you'll also need to change this to `<Route path="/volunteers" component={VolunteersForm} />`